rotkehlchen/tests/conftest.py

```diff
@@ -28,6 +28,7 @@
     from vcr import VCR
 
 
+os.environ['POLARS_ALLOW_FORKING_THREAD'] = '1'
```

Comment:
Why do you need this?

Reply:
It appears every time that you run a test and since it is big it's annoying. We are aware of it so it is fine to silence it 
![imagen](https://github.com/user-attachments/assets/8e13eae9-caa9-44af-b128-b81d9741fd18)

Reply:
I will add a comment here in my PR